logs/app-error.log

```diff
@@ -19,3 +19,13 @@ java.lang.NumberFormatException: For input string: "dfds"
 	at java.lang.Integer.valueOf(Integer.java:992) ~[?:?]
 	at com.ruslan.ui.MenuController.run(MenuController.java:37) [classes/:?]
 	at TestBookService.main(TestBookService.java:14) [classes/:?]
+2023-11-18T07:19:24.545+0100 ERROR c.r.D.ObjectFactory [main] java.lang.NoSuchMethodException: com.ruslan.database.DAO.BookRepository.<init>()
```

Comment:
put it in gitignore and other redundant files as well

Reply:
fixed